samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
```

Comment:
I wonder if we should actually select or imply these commands (I think it would be good provide complete configuration by default and allow user to experiment).

Reply:
Sure, we can have these configuration options implied so it is possible for the user to explicitly disable them